src/wallet/components/FundingError.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Button from '../../components/Button';
+
+interface Props {
+  message: string;
+  tryAgain: () => void;
+}
+
+export default class FundingError extends React.PureComponent<Props> {
+  render() {
+    return <div>
+    <div>{this.props.message}</div>
+    <Button onClick={this.props.tryAgain}>Try Again</Button>
```

Comment:
Not important for this PR but in the long term we should have a "Give up" option, that would send a `FUNDING_FAILED` event to the app and return you there.

Reply:
Yup, I agree. I figured we'd want that but I decided not to tackle it now.